src/Spryker/Client/RabbitMq/Model/Connection/ConnectionCreator/ConnectionCreator.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace Spryker\Client\RabbitMq\Model\Connection\ConnectionCreator;
+
+use Generated\Shared\Transfer\QueueConnectionTransfer;
+use PhpAmqpLib\Connection\AMQPStreamConnection;
+use Spryker\Client\RabbitMq\Dependency\Client\RabbitMqToStoreClientInterface;
+use Spryker\Client\RabbitMq\Model\Connection\Connection;
+use Spryker\Client\RabbitMq\Model\Connection\ConnectionInterface;
+use Spryker\Client\RabbitMq\Model\Helper\QueueEstablishmentHelperInterface;
+use Spryker\Client\RabbitMq\RabbitMqConfig;
+
+class ConnectionCreator implements ConnectionCreatorInterface
```

Comment:
I think `ConnectionFacatory` or `Builder` is a better name for instantiating the connection